src/accumulator.jl

```diff
@@ -92,7 +90,9 @@ Decrements the count for `x` by `v` (defaulting to one)
 dec!(ct::Accumulator, x, v::Number) = (ct[x] -= v)
 dec!(ct::Accumulator{T,V}, x) where {T,V} = dec!(ct, x, one(V))
 
-#TODO: once we are done deprecating `pop!` for `reset!` then add `pop!` as an alias for `dec!`
```

Comment:
I assume it was decided at some point that `pop!` is not being deprecated so this TODO is incorrect and that is why it is deleted and disobeyed?

Reply:
When `pop!` becomes an alias for `dec!` as seen here, then this will be happening:

```
julia> a = Accumulator(:k=>1); dec!(a, :k); dec!(a, :k)
-1
```

Just making sure that this is the desired behavior. Asking because I would have expected `KeyError` when the count goes negative.